src/TutorialTemplate.js

```diff
@@ -1,48 +1,71 @@
-import React from 'react';
-import styled from 'styled-components/macro';
-import meditation from './assets/image-meditation.jpg'
+import React from 'react'
+import styled from 'styled-components/macro'
+import data from './data/content.json'
 
-
-export default function TutorialTemplate (){
-    return (
-    <TutorialWrapper>
-        <img src={meditation} alt=""/>
-        <h2>5 Minuten of...</h2>
-        <p>Lorem ipsum dolor sit amet consectetur Quos accusamus libero minus ullam, beataelibero minus ullam, beatae possimus nostr</p>
-    </TutorialWrapper>
-    )   
+export default function TutorialTemplate() {
+  return (
+    <>
+      {data.map(
+        ({
+          headline,
+          title,
+          image,
+          description1,
+          description2,
+          description3,
+          id,
+        }) => (
+          <TutorialWrapper key={id}>
+            <img src={image} alt="" />
+            <h2>{headline}</h2>
+            <h3>{title}</h3>
+            <p>{description1}</p>
+            <p>{description2}</p>
+            <p>{description3}</p>
+          </TutorialWrapper>
+        )
+      )}
+    </>
+  )
 }
 
 const TutorialWrapper = styled.div`
-    display: grid;
-    border: 1px solid var(--primary-border);
-    background-color: var(--secendary-background); 
-    box-shadow:  20px 20px 60px var(--primary-shadow),
-                -10px -10px 50px var(--secendary-shadow);
-    margin-left: 60px;
-    margin-top: 30px;
-    padding:0 15px 20px 15px;
-    place-items: center;
-    width: 70%;
+  display: grid;
+  border: 1px solid var(--primary-border);
+  background-color: var(--secondary-background);
+  box-shadow: 20px 20px 60px var(--primary-shadow),
+    -10px -10px 50px var(--secondary-shadow);
+  margin-top: 30px;
+  padding: 0 15px 20px 15px;
+  place-items: center;
+  width: 70%;
 
-    img {
-        border-radius: 2px;
-        margin-top: 25px;
-        width: 80%;
-    }
+  img {
+    border-radius: 2px;
+    margin-top: 25px;
+    width: 80%;
+  }
 
-    h2 {
-        font-size: 1em;
-        font-weight: 500;
-        text-align: center;
-        text-transform: uppercase;
-    }
+  h2 {
+    font-size: 1em;
+    font-weight: 500;
+    text-align: center;
+    text-transform: uppercase;
+  }
+  h3 {
+    font-size: 0.8em;
+    font-weight: 400;
+    margin-top: 0;
+    text-align: center;
+    text-transform: uppercase;
+  }
 
-    p {
-        font-size: 0.8em;
-        font-weight: 200;
-        margin: 0;
-        padding: 0 10px 0 10px;
-        text-align: center;
-    }
-`
\ No newline at end of file
+  p {
+    font-size: 0.8em;
+    font-weight: 200;
+    hyphens: auto;
+    margin: 10px 0 0 0;
+    padding: 0 10px 0 10px;
+    text-align: center;
```

Comment:
Does `text-align: center;` work without being a `display: flex;` for example? Just wondering? :)